apps/rush-lib/src/logic/taskRunner/TaskRunner.ts

```diff
@@ -443,4 +443,21 @@ export class TaskRunner {
     }
   }
 
+  /**
+   * Remove trailing blanks, and all middle lines if text is large
+   */
+  private _abridgeTaskReport(text: string): string {
```

Comment:
Can you add a unit test for this case as well?

Reply:
I thought the snapshots would capture how leading & trailing blanks are handled, this but it was not the case. So I added a separate test to explicitly verify leading blanks are preserved and trailing blanks are trimmed. Is that what you meant here?

(also added a bunch of snapshots)